app/views/shared/_upcoming_events.html.erb

```diff
@@ -0,0 +1,44 @@
+<section>
+    <h1> Upcoming Events </h1>
+    <h3> See What's Next! </h3>
```

Comment:
I don't think there should be an h1 here (because you should only really have one of them on a page), but I'm sure that we'll be doing a complete refactor of our views on Friday.

Both of these headings have a space before and after them, which is a bit strange, so I think you should remove those spaces.